index.js

```diff
@@ -155,6 +155,7 @@ function Auth0 (options) {
     facebook: this._phonegapFacebookLogin
   };
   this._useCordovaSocialPlugins = false || options.useCordovaSocialPlugins;
+  this._sendClientInfo = (typeof options.sendSDKClientInfo === 'undefined') ? true : options.sendSDKClientInfo;
```

Comment:
```
null == null // true
null == undefined // true
null == false // false
null == true // false
```

`this._sendClientInfo = null != options.sendSDKClientInfo ? options.sendSDKClientInfo : true;`